audit-service/src/unit-tests/java/uk/gov/gchq/palisade/service/audit/service/LoggerAuditServiceTest.java

```diff
@@ -44,98 +52,100 @@
     @Captor
     ArgumentCaptor<String> errorCaptor;
 
-    private static LoggerAuditService auditService;
+    private LoggerAuditService auditService;
```

Comment:
chuck in some .as("") descriptions in this class

Reply:
Added/updated some `as()` descriptions